test/dummy/app/controllers/pdfs_controller.rb

```diff
@@ -6,34 +6,22 @@ def show
     respond_to do |format|
       format.html
       format.pdf {
-        pdf = render_pdf(
-          layout: "pdf",
-          pdf_options: {
+        render ferrum_pdf: {
             display_header_footer: true,
             header_template: FerrumPdf::DEFAULT_HEADER_TEMPLATE,
             footer_template: FerrumPdf::DEFAULT_FOOTER_TEMPLATE
-          }
-        ) do |browser, page|
```

Comment:
This block is still doable, but only with `FerrumPdf.render_pdf()` calls directly.